Packs/Druva/Playbooks/Druva-Ransomware-Response-User-resource-URL.yml

```diff
@@ -1,546 +0,0 @@
-id: Druva-Ransomware-Response-User-resource-URL
-version: -1
```

Comment:
Please do not delete existing content items. You can deprecate it instead: https://xsoar.pan.dev/docs/reference/articles/deprecation-process-and-hidden-packs#how-to-deprecate-a-playbook

Reply:
This was never meant for publishing.
Hence, we need to delete this.